src/styles/blocs/card.scss

```
@@ -0,0 +1,125 @@
+
+.catalog {
+  padding-top: 46px;
+  // display: flex;
+  // justify-content: center;
+  // padding: 46px 40px;
+  // margin: auto;
```

```suggestion
.catalog {
  padding-top: 46px;
  // display: flex;
  // justify-content: center;
  // padding: 46px 40px;
  // margin: auto;
```
leave such comments in code is not a good idea )